### ISSUE TITLE
cannot export data from yeastmine, ratmine, mousemine

### ISSUE THREAD

**Issue by martenson** [galaxyproject/galaxy#3493]
Tested on Main. Possibly affects other sources too. Reported by sangram_twit_twitter on gitter.

After clicking export it seems to open url without the tool id such as `https://usegalaxy.org/tool_runner` or just plain nonsense like `http://www.mousemine.org/mousemine/usegalaxy.org/tool_runner`

![screenshot 2017-01-26 12 23 34](https://cloud.githubusercontent.com/assets/1814954/22342416/591cd35a-e3c2-11e6-9ad6-60911083494a.png)

**Comment by martenson** [comment]
@jennaj can you please test this to confirm?

**Comment by jennaj** [comment]
Yes, this is the error on these site's side when attempting to send data to Galaxy. I tested a few formats/options - all result in this error. 

I changed the target URL from http://main.g2.bx.psu to be http://usegalaxy.org as part of that testing. No impact (expected).

_However_ - that outdated URL does suggest that it has been a while since this external service has been reviewed on their side if the older URL is in use. Maybe changes to galaxy are no longer compatible with the other sites. Perhaps we can help them align - or we make changes to better align with them.

I thought this was a known - the tools haven't been working for a while and the reason why at the time to not fix/investigate I can't recall. But can search for earliest reference if matters.

**Comment by martenson** [comment]
@jennaj If they don't work they should be hidden from the toolpanel.

**Comment by jennaj** [comment]
Fairly certain that was considered at one point. @blankenberg would probably more know about all of this than me.

**Comment by martenson** [comment]
@jennaj Can you please contact them and ask whether they can try fixing it on their side. I do not think we broke it.

**Comment by jennaj** [comment]
Ok, I have the contact/connection

**Comment by chewyalvin** [comment]
no updates on this issue? I am trying to export data from Yeastmine to Galaxy

**Comment by chewyalvin** [comment]
This location seems to work: https://usegalaxy.org/
